addons/website_mass_mailing/models/link_tracker.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models
+
+from werkzeug import urls
+
+
+class LinkTracker(models.Model):
+    _inherit = ['link.tracker']
+
+    def _compute_short_url_host(self):
```

Comment:
Dependency on mass_mailing_id is missing (@api.depends).

Reply:
Ok might be better indeed, I modified it